packages/swapper/src/swappers/ThorchainSwapper/evm/utils/getThorTxData.ts

```diff
@@ -33,8 +34,8 @@ export const getThorTxInfo = async ({
   if (maybeInboundAddress.isErr()) throw maybeInboundAddress.unwrapErr()
   const inboundAddress = maybeInboundAddress.unwrap()
 
-  const router = inboundAddress.router
-  const vault = inboundAddress.address
+  const router = getAddress(inboundAddress.router as string)
+  const vault = getAddress(inboundAddress.address as string)
 
   if (!router) {
```

Comment:
non blocking suggestion - this changes error behavior slightly. We can move below our `if (!router)` check (probably change to `if (!inboundAddress.router)`, then have our custom error message and no more need to cast

Reply:
I prefer not to - this was added by gomes's review feedback and strictly isnt necessary but I'd prefer not to add more change because its really not related to the PR